### ISSUE TITLE
[HIPRI] Adjustments to new database PLAAF radar data

### ISSUE THREAD

**Issue by yUsie108** [PygmalionOfCyprus/cmo-db-requests#4906]
### DB Selector

DB3K

### Affected DBID(s)

Sensor 5949

### Summary of Changes

- [ ] The J-20 radar data is adjusted to the radar peak power of 150w and gain of 50dB
- [ ] The radar value of J-16, J-15B and other advanced lateral models is adjusted to the radar peak power of 150w and gain of 45dB.

In the recently updated beta version, I found that the J-20's radar had been tweaked, and after testing, I found that the performance was not as good as usual, mainly due to radar peak power and lightning handling gain/loss issues.
Before talking about how to modify the J-20 radar, let me tell you a little about the LPI radar in the CMO. In CMO, whether a radar is an LPI radar is actually determined by radar peak power and processing gain/loss. We all know that the dB value is the logarithm of a linear number (the specific formula I discuss is listed in the source code below).
Therefore, if the detection capability of the radar is enhanced by a hard direct increase in power, that is, the peak power of the radar is high and the gain is low, then the ESM device can easily listen for the radar's starting signal, leaving the target exposed. On the contrary, radar with low peak power and high gain is extremely difficult to be detected by conventional ESM equipment because the radar gain can use high gain to achieve the detection effect of equivalent high-power radar.
To sum up, in CMO, LPI radars achieve this effect not directly by endowing LPI characteristics, but by adjusting the relationship between radar peak power and radar gain. Therefore, the requirement here is to adjust the peak power of the J-20's KLJ-5(1475) radar to 150w and the gain to 50dB. At the same time, the remaining PLAAF LPI radars tuned to high power and low gain were modified to low power and high gain LPI radar models.

### Sources

![image](https://github.com/user-attachments/assets/839ef18f-6877-44e8-b989-d5964b416e6f)
↑ This is the formula for converting linear numbers to dB numbers, where N is the dB value to be substituted
Database itself

**Comment by PygmalionOfCyprus** [comment]
Janes (which admittedly I take with a grain of salt) says the Type 1475 is 100kW: https://www.janes.com/osint-insights/defence-news/defence/pla-says-j-20-tracking-foreign-military-aircraft-in-east-china-sea-adiz

**Comment by PygmalionOfCyprus** [comment]
I don't necessarily disagree, but how do we reconcile this w/ Janes reporting that claims the Type 1475 is a 100kW radar? https://www.janes.com/osint-insights/defence-news/defence/pla-says-j-20-tracking-foreign-military-aircraft-in-east-china-sea-adiz

A 150w/50 setting would make it stealthier than the APG-77. I just don't think that the Chinese have achieved that level of LPI with an early-generation LPI AESA.

**Comment by PygmalionOfCyprus** [comment]
Closing in favor of #4935
